File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <stdio.h>

int main(void)
{
	int array[9];
	int i, index;
    int max = 0;

	for (i = 0; i < 9; i++)
	{
		scanf("%d\n", &array[i]);

		if (array[i] > max)
		{
			max = array[i];
			index = i;
		}			
	}

	printf("%d\n%d", max, index + 1);
}

File: 백준/Bronze/2566. 최댓값/최댓값.py
max_num = -1
max_row = 0
max_col = 0

for i in range(9):
  nums = list((map(int, input().split())))
  x = max(nums)
  if max_num < x:
    max_num = x
    max_row = i + 1
    max_col = nums.index(x) + 1

print(max_num)
print(max_row, max_col)